test/state/state.cpp

```diff
@@ -136,11 +136,26 @@ std::variant<int64_t, std::error_code> validate_transaction(const Account& sende
 }
 
 void finalize(State& state, evmc_revision rev, const address& coinbase,
-    std::optional<uint64_t> block_reward, std::span<Withdrawal> withdrawals)
+    std::optional<uint64_t> block_reward, std::span<Withdrawal> withdrawals,
+    std::span<Ommer> ommers)
 {
     // TODO: The block reward can be represented as a withdrawal.
     if (block_reward.has_value())
+    {
         state.touch(coinbase).balance += *block_reward;
+        if (!ommers.empty())
+        {
+            state.touch(coinbase).balance += *block_reward * ommers.size() / 32;
+
+            for (const auto& ommer : ommers)
+            {
+                uint256 block_reward_256 = *block_reward;
+                block_reward_256 = block_reward_256 * (8 - ommer.delta) / 8;
```

Comment:
Just initialize once to make it const
```suggestion
                const auto block_reward_256 = u256{*block_reward} * (8 - ommer.delta) / 8;
```

Reply:
Outdated.